app/js/controllers/resultat.js

```diff
@@ -26,7 +26,8 @@ angular.module('ddsApp').controller('ResultatCtrl', function($scope, $rootScope,
         $scope.noDroits = _.isEmpty($scope.droits.prestationsNationales) && _.isEmpty($scope.droits.partenairesLocaux);
     })
     .catch(function(error) {
-        $scope.error = (error && error.data) || error || true;
+        var errorProxy = (error && error.data) || error || true;
```

Comment:
Poor naming. You're describing what you are using the object for rather than what it is, but its scope is broader than the line right after.

I would not be shocked by not using a temporary variable at all, even more since:

- `(error && error.data) || error` is equivalent to `error && error.data`.
- Using `JSON.stringify` means the encoded `error` value will yield a truthy, even if the original `error` was a falsy (except if it was `undefined`; it would be very weird for `undefined` to be explicitly thrown, but reading the source promises should make it clear whether this is possible).